config/delta-producer/publication-graph-maintainer/config.json

```
@@ -137,5 +137,29 @@
     "publisherUri": "http://data.lblod.info/services/delta-producer-publication-graph-maintainer-vendor-management",
     "filesGraph": "http://redpencil.data.gift/id/deltas/producer/vendor-management-deltas",
     "account": "http://data.lblod.info/foaf/account/id/0a3d5be8-6f6b-4c4e-9069-9b7c57a35df3"
-  }
+  },
+    "worship-services-sensitive": {
+      "errorCreatorUri": "http://lblod.data.gift/services/delta-producer-publication-graph-maintainer-worship-services-sensitive",
+      "exportConfigPath": "/config/worship-services-sensitive/export.json",
+      "filesPath": "/worship-services-sensitive/files",
+      "jobsGraph": "http://mu.semte.ch/graphs/organizations/141d9d6b-54af-4d17-b313-8d1c30bc3f5b/LoketAdmin",
+      "relativeFilePath": "deltas/worship-services-sensitive",
+      "publicationGraph": "http://redpencil.data.gift/id/deltas/producer/loket-worship-services-sensitive",
+      "healingJobOperation": "http://redpencil.data.gift/id/jobs/concept/JobOperation/deltas/healingOperation/worship-services-sensitive",
+      "initialPublicationGraphSyncJobOperation": "http://redpencil.data.gift/id/jobs/concept/JobOperation/deltas/initialPublicationGraphSyncing/worship-services-sensitive",
+      "queuePollInterval": 3000,
+      "useVirtuosoForExpensiveSelects": true,
+      "skipMuAuthInitialSync": true,
+      "skipMuAuthHealing": true,
+      "useFileDiff": true,
+      "healingInitialBatchSizeInsert": 1000,
+      "updatePublicationGraphSleep": 200,
+      "serveDeltaFiles": true,
+      "deltaInterval": 10000,
+      "deltaPath": "/worship-services-sensitive/delta",
+      "loginPath": "/worship-services-sensitive/login",
+      "publisherUri": "http://data.lblod.info/services/delta-producer-publication-graph-maintainer-ws-sensitive",
+      "filesGraph": "http://redpencil.data.gift/id/deltas/producer/worship-services-sensitive-deltas",
+      "account": "http://data.lblod.info/foaf/account/id/0a3d5be8-6f6b-4c4e-9069-9b7c57a35df3"
```

This should be the right account:
```suggestion
      "account": "http://data.lblod.info/foaf/account/id/f185b63e-5bd9-40a5-8906-4ac1fb5a8db7"
```

If I check in the database locally, it is connected to `http://data.lblod.info/foaf/group/id/c59564df-8a3e-4834-bda4-ceca83e1dad7`, which is the "worship-services-sensitive-deltas" group.